dali/test/python/test_plugin_manager.py

```diff
@@ -95,11 +95,19 @@ def test_load_existing_but_not_a_library(self):
     def test_load_custom_operator_plugin(self):
         with self.assertRaises(AttributeError):
             print(ops.CustomDummy)
-        plugin_manager.load_library(test_bin_dir + "/libcustomdummyplugin.so")
+        try:
```

Comment:
how about wrapping this in a function to avoid repeating it 3 times?

Reply:
Done